README.md

```diff
@@ -16,21 +16,38 @@ Follow [these](#additional-documentation-sources) instructions to enable optiona
 
 ## Installation
 
-### Arch Linux / Manjaro ([AUR](https://aur.archlinux.org/packages/wikiman/))
+### Arch Linux
+
+**Wikiman** is available in the [community repository](https://archlinux.org/packages/community/any/wikiman/) and can be installed via [pacman](https://wiki.archlinux.org/title/Pacman):
```

Comment:
I would reword this to be more consise, akin to [FreeBSD](https://github.com/filiparag/wikiman#freebsd) one:
```markdown
Install from Arch Linux's [community](https://archlinux.org/packages/community/any/wikiman/) repository:
```

Reply:
49804299ba104136ff5eea065d8a67310ebdae06